Failed test in release pipeline

The last release failed on Azure. I tried 3 times. Those are not flacky tests.

There are few things in the logs, not sure which one make the job fail:

```
23 07 2025 08:24:33.104:INFO [Chrome Headless 79.0.3945.0 (Linux x86_64) | clone | clone from git-http-mock-server]: Success: 94 ms
Test Num: 493
Elapsed Time: 0:55:444 min/sec/ms
Chrome Headless 79.0.3945.0 (Linux x86_64) LOG: ApiError{syscall: '', errno: 95, code: 'ENOTSUP', path: undefined, stack: 'Error
    at new ApiError (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:28438:22)
    at UnlockedOverlayFS.symlink (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:31149:8)
    at http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:38311:20
    at Mutex.lock (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:37970:5)
    at OverlayFS.symlink (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:38310:14)
    at FS.symlink (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:29707:31)
    at Object._fsMock.<computed> [as symlink] (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:30014:28)
    at http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:20192:79661
    at new Promise (<anonymous>)
    at p.<anonymous> ([http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:20192:79501)',](http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:20192:79501)%27,) message: 'Error: ENOTSUP: Operation is not supported.'}
23 07 2025 08:24:42.191:INFO [Chrome Headless 79.0.3945.0 (Linux x86_64) | clone | should set up the remote tracking branch by default]: ApiError{syscall: '', errno: 95, code: 'ENOTSUP', path: undefined, stack: 'Error
    at new ApiError (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:28438:22)
    at UnlockedOverlayFS.symlink (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:31149:8)
    at http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:38311:20
    at Mutex.lock (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:37970:5)
    at OverlayFS.symlink (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:38310:14)
    at FS.symlink (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:29707:31)
    at Object._fsMock.<computed> [as symlink] (http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:30014:28)
    at http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:20192:79661
    at new Promise (<anonymous>)
    at p.<anonymous> ([http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:20192:79501)',](http://localhost:9876/base/__tests__/index.webpack.js?41dc105870a627aaa7a37aa2909e23b5d3bd8009:20192:79501)%27,) message: 'Error: ENOTSUP: Operation is not supported.'}
```

```
Success! index.js is fully tree-shakeable
nps is executing `build.docs` : node ./__tests__/__helpers__/generate-docs.cjs
nps is executing `build.size` : (cross-env BUNDLEWATCH_GITHUB_TOKEN='***' CI_REPO_OWNER='isomorphic-git' CI_REPO_NAME='isomorphic-git' CI_COMMIT_SHA='undefined' CI_BRANCH='undefined' CI_BRANCH_BASE='undefined' bundlewatch) || echo "Optional command 'cross-env BUNDLEWATCH_GITHUB_TOKEN=\'***\' CI_REPO_OWNER=\'isomorphic-git\' CI_REPO_NAME=\'isomorphic-git\' CI_COMMIT_SHA=\'undefined\' CI_BRANCH=\'undefined\' CI_BRANCH_BASE=\'undefined\' bundlewatch' failed".
[ERROR] GitHubService HTTP_422 :: No commit found for SHA: undefined
[INFO] Retrieving comparison
[ERROR] Unable to fetch fileDetails for baseBranch=undefined from https://service.bundlewatch.io/store code=ERR_BAD_REQUEST
[INFO] undefined is not a branch to track, no results saved
[ERROR] GitHubService HTTP_422 :: No commit found for SHA: undefined
```


```
23 07 2025 09:17:41.288:ERROR [launcher.browserstack]: Can not start android (android 10.0)
  Error: Validation Failed - `os_version` [BROWSERSTACK_DEPRECATED_DEVICE_ERROR] This is to inform you that we have deprecated Google Pixel 4-10.0 from 18 July 2025. You will no longer be able to run tests on Google Pixel 4-10.0 on BrowserStack. For more details, please check our device deprecation policy https://www.browserstack.com/question/790.  We strongly recommend you update your test scripts. For the current list of supported devices on BrowserStack, please check here - https://www.browserstack.com/list-of-browsers-and-platforms/automate
```

--- a/karma.conf.cjs
+++ b/karma.conf.cjs
@@ -105,9 +105,9 @@ module.exports = function (config) {
       bs_android_chrome: {
         base: 'BrowserStack',
         os: 'android',
-        os_version: '10.0',
+        os_version: '11.0',
         browser: 'android',
-        device: 'Google Pixel 4',
+        device: 'Google Pixel 5',
         real_mobile: true,
         captureTimeout: 5 * 60 * 1000, // defaults to 120 ms
         timeout: 1000,                 // defaults to 300 ms

